
{% load staticfiles %}

<div class="text-justify">
<p>
<h2>Contents</h2>
<a class="g" href="#target">The target view</a><br />
<a class="g" href="#mirna">The microRNA view</a><br />
<a class="g" href="#pathway">The pathway view</a><br />

</p>

<a name="target"></a>
<h3>The target view</h3>
<p>
This is the view for one particular target gene. In this example we are looking at target gene <a class="t1" href="{{ target_url }}Pdpk1">Pdpk1</a>. The first tab shows all miRNAs that are either confirmed or predicted to be targeting <a class="t1" href="{{ target_url }}Pdpk1">Pdpk1</a>. We also are able to see the sources.
</p>
<p>
<img src="{% static 'irn/desc/target1.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<p>
The next two tabs are similar to the miRNA views tabs but for piRNAs and lncRNAs that interact with the target gene. The next two tabs show pathway and GO-terms that are associated with the target gene. In the pathway view, one can find links to IRNdb individual pathway view pages that show all targets of ncRNAs in the pathway (see below "The pathway view").
</p>
<p>
The <b>Experiment-tab</b> shows immunologically themed experiments in which the target gene is either down- or up-regulated. We also find a link to <a class="g" href="https://www.broadinstitute.org/gsea/msigdb/">MSigDB</a> where the information was extracted from and a link to the original publication and dataset if available.
</p>
<p>
<img src="{% static 'irn/desc/target2.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<p>
The final <b>M.tb-tab</b> shows the expression and fold-changes (of available) of the gene in the M.tb dataset for different stimulations <a href="http://www.ncbi.nlm.nih.gov/pubmed/25678556" alt="Pubmed">PMID:25678556 <img src="{% static 'irn/pubmed-small.png' %}" title="Transcribed enhancers lead waves of coordinated transcription in transitioning mammalian cells. Science. 2015 Feb 27;347(6225):1010-4." /></a>.
</p>
<p>
<img src="{% static 'irn/desc/target3.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<a name="mirna"></a>
<h3>The microRNA view</h3>

<p>
Here, we have look at the miRNA view of an individual miRNA found either through the <a class="g" href="{{ browse_url }}?type=mirna">browse/search</a> function or through a target view of the site. In this example, we look at the miRNA <a class="m1" href="{{ mirna_url }}mmu-miR-223-3p">mmu-miR-223-3p</a>. The first image shows the "Targets"-tab of the miRNA view. Here one can investigate the experimentally supported and predicted targets of the miRNA, considering only mouse or mouse+human immune genes.
</p>
<p>
<img src="{% static 'irn/desc/mirna1.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<p>
The next tab is the <b>"Pathways"-tab</b> of the miRNA view. Here one can investigate the pathways that the targets of the miRNA are part of.<br><br>
<img src="{% static 'irn/desc/mirna2.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>
<p>
On the <b>"GO"-tab</b> of the miRNA view one can investigate the Gene Ontology terms that the targets of the miRNA are associated with.
</p>
<p>
<img src="{% static 'irn/desc/mirna3.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<p>
The <b>TFBS-tab</b> shows transcription factor binding sites based on ChIP experiments that we collected. The process to derive these is a bit more involved. First, we used <a class="g" href="http://promirna.molgen.mpg.de/">PROmiRNA</a> [<a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/23958307">1</a>] to identify promoter regions of pri-miRNAs (based on <a href="http://www.mirbase.org/" class="g">miRBase</a>) using data from [<a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/25678556" alt="Pubmed">2</a>,<a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/26117544" alt="Pubmed">3</a>]. Next, we mapped 1000nt upstream of the CAGE promoter start ChIP data from several sources (see "Resources"). A mature miRNA can have more then one pri-miRNA with several locations in the genome. Furthermore, each pri-miRNA can have several associated CAGE promoters. We show for each pri-miRNA an individual table.
</p>
<p>
<img src="{% static 'irn/desc/mirna4.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<p>
Finally, on the <b>M.tb-tab</b> of the miRNA view, we show for <i>targets</i> the fold-change values of a particular experiment [<a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/25678556" alt="Pubmed">2</a>,<a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/26117544" alt="Pubmed">3</a>]. Here, the log2(fold-change) values of the experiment are visualised. We only show the lowest log2(fold-change) value per stimulation post TB infection over all time-points measured. The idea is that miRNAs down-regulate their target expression, thus one might be interested in down-regulated targets. Expression for pri-miRNAs of the mature miRNA is shown as the summation of all its associated promoters based on the strategy explained above. Should pri-miRNAs be missing for a mature miRNA, then we were unable to associate promoters to the pri-miRNAs.
</p>
<p>
<img src="{% static 'irn/desc/mirna5.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>    

<a name="pathway"></a>
<h3>The pathway view</h3>
<p>
Here, we have look at the pathway view. In this example, we look at the KEGG pathway <a class="m1" href="{{ pw_url }}mmu05164">Influenza A</a>. We find some general inforamtion about the pathway and the numbers of miRNAs, piRNAs, and lncRNA that target genes in the pathway. In addition, one can find association-maps of target genes to RNAs. Finally, we show the pathway map itself.
</p>
<p>
<img src="{% static 'irn/desc/pathway1.png' %}" style="vertical-align: middle; border:1px solid #021a40;" alt="" / >
</p>

<h2>References</h2>
<ol>
  <li><a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/23958307">PROmiRNA: a new miRNA promoter recognition method uncovers the complex regulation of intronic miRNAs. Genome Biol. 2013 Aug 16;14(8):R84.</a></li>
  <li><a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/25678556" alt="Pubmed">Transcribed enhancers lead waves of coordinated transcription in transitioning mammalian cells. Science. 2015 Feb 27;347(6225):1010-4.</a></li>
  <li><a class="g" href="http://www.ncbi.nlm.nih.gov/pubmed/26117544">Redefining the transcriptional regulatory dynamics of classically and alternatively activated macrophages by deepCAGE transcriptomics. Nucleic Acids Res. 2015 Aug 18;43(14):6969-82.</a></li>
</ol> 

</div>